src/content/docs/jellyfin-integration/installing-shokofin.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Installing Shokofin
+sidebar:
+  order: 1
+Description: Information on how to install and configure Shokofin (Jellyfin)
+---
+
+import EasyTable from "../../../components/EasyTable/EasyTable";
+
+**Shokofin** is an anime metadata provider plugin for **Jellyfin** that works in tandem with **Shoko Server** to provide
+metadata tailored to your personal anime collection. For information on what Shoko Server is and the benefits that
+can come from using it to identify and manage your collection, visit our [Official Website](https://shokoanime.com/).
+
+## Requirements
+
+Shokofin is not meant to be used stand-alone and requires Shoko Server to be installed, configured, and running before
+it can pull in metadata specific to your library. If you haven't done so, [Download Shoko Server](https://shokoanime.com/downloads/)
+from our main website and use the [Installing Shoko Server](../../../getting-started/installing-shoko-server) guide to
+properly install and configure Shoko Server.
+
+Shokofin requires Shoko Server to already be setup and in use, and that the directories/folders you intend to
+use in Jellyfin are fully indexed (and optionally managed) by Shoko Server. Without doing so beforehand, Shokofin will be
```

Comment:
so, because of the new and shiny Virtual File System (VFS) feature is now implemented, then the requirements for the library to be fully indexed can be dropped as long as the VFS is used.